pkg/proxy/ipvs/proxier.go

```diff
@@ -650,6 +650,8 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 			if err != nil {
 				klog.Warningf("Failed to load kernel module %v with modprobe. "+
 					"You can ignore this message when kube-proxy is running inside container without mounting /lib/modules", module)
+			} else {
+				bmods = append(bmods, module)
```

Comment:
Logically this seems correct. The problem is that `CanUseIPVSProxier` returns false (hence we set a nil ipvs interface) because it doesn't account for modules we load at startup so it errors [here](https://github.com/kubernetes/kubernetes/blob/release-1.18/pkg/proxy/ipvs/proxier.go#L712-L714). 

Though we should use a new variable for this (`loadedMods` or similar).

Reply:
can we put it to `mods` directly?
as `mods = append(mods, module)` avoid adding a []string, which one do you think better?

Reply:
not add a new var `loadedMods` because add it to mods seems more concise :)
I'm sorry to handle other comments tomorrow morning.

Reply:
Though not as concise, I think adding a `loadedMods` var will be better for readability.

Reply:
> Though not as concise, I think adding a `loadedMods` var will be better for readability.

Done, PTAL